### PR TITLE
Update order screen: Part 2

### PR DESCRIPTION
**Pull request by nhokzminh698**
- Changed the orders screen so that instead of showing table details it is now in a square box with the necessary information like order number, costumer name and subtotal. 
- Click pay in the square would lead to the payment screen of that order to see the item details.
- Make an order in the ordering page then it should leads to the orders screen to see them squares (technically not square because I wanted it to spill all over the page)!

### CODE REVIEW

File: app/frontend/private/pos/orders/script.js [hunk 1]
```diff
@@ -14,18 +14,18 @@ fetch("/api/orders")
     })
     .then((body) => {
         for (let order of body) {
-            const tr = document.createElement("tr");
+            const squareButton = document.createElement("button");
+            squareButton.classList.add("order-button");
```
Comment (gitnamehere) [inline app/frontend/private/pos/orders/script.js:18]:
```suggestion
            const squareButton = document.createElement("a");
            squareButton.href = `pos/payment?id=${order.id}`
```

File: app/frontend/private/pos/orders/script.js [hunk 2]
```diff
@@ -14,18 +14,18 @@ fetch("/api/orders")
...
-                </td>
+            squareButton.innerHTML = `
+                <div class="order-info">
+                    <div>Order #${order.id}</div>
+                    <div>Customer: Michael</div>
+                    <div>Subtotal: $${order.subtotal}</div>
+                    <div class="order-pay">
+                        <a href="/pos/payment?id=${order.id}" class="btn btn-primary">Pay</a>
+                    </div>
+                </div>
```
Comment (gitnamehere) [inline app/frontend/private/pos/orders/script.js:28]:
```suggestion
                <div class="order-button btn btn-light">
                    <div>Order #${order.id}</div>
                    <div>Michael</div>
                    <div>$${order.subtotal}</div>
                </div>
```

File: app/frontend/private/pos/orders/styles.css [hunk 3]
```diff
@@ -30,3 +30,28 @@
     font-size: 24px;
     font-weight: 700;
 }
+
+#orders {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 60px;
+    padding: 50px;
+}
```
Comment (gitnamehere) [inline app/frontend/private/pos/orders/styles.css:39]:
```suggestion
#orders {
    display: flex;
    margin-top: 16px;
    flex-wrap: wrap;
    gap: 16px;
}
```

File: app/frontend/private/pos/orders/styles.css [hunk 4]
```diff
@@ -30,3 +30,28 @@
...
+
+.order-button {
+    width: 250px;
+    height: 200px;
```
Comment (gitnamehere) [inline app/frontend/private/pos/orders/styles.css:43]:
```suggestion
    width: 160px;
    aspect-ratio: 1/1;
```

File: app/frontend/private/pos/orders/styles.css [hunk 5]
```diff
@@ -30,3 +30,28 @@
...
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: #f8f9fa;
+    font-size: 16px;
+    text-align: center;
+    padding: 10px;
+    transition: background-color 0.3s;
+}
+
```
Comment (gitnamehere) [inline app/frontend/private/pos/orders/styles.css:54]:
```suggestion

#orders a {
    text-decoration: none;
}

```

**Review by gitnamehere — CHANGES_REQUESTED** [review]
looks good, though I've added some comments for a potential design in the order buttons because having a pay button in each order tile is redundant

**Comment by nhokzminh698** [comment]
> looks good, though I've added some comments for a potential design in the order buttons because having a pay button in each order tile is redundant

yessir lemme look and try them out

File: app/frontend/private/pos/orders/script.js (hunk 1 above)
Comment (nhokzminh698) [reply app/frontend/private/pos/orders/script.js:18]: I tried this but it goes to http://localhost:3000/pos/orders/pos/payment?id=3

Comment (gitnamehere) [reply app/frontend/private/pos/orders/script.js:18]: oops, should be `/pos/payment?id${order.id}`

**Review by gitnamehere — APPROVED** [review]

**Pull request merged**